clicommand/annotate.go

```diff
@@ -55,9 +55,10 @@ type AnnotateConfig struct {
 	Job     string `cli:"job" validate:"required"`
 
 	// Global flags
-	Debug   bool   `cli:"debug"`
-	NoColor bool   `cli:"no-color"`
-	Profile string `cli:"profile"`
+	Debug   bool         `cli:"debug"`
+	NoColor bool         `cli:"no-color"`
+	Experiments []string `cli:"experiment" normalize:"list"`
```

Comment:
I see the global flags, including `Experiments`, are handled by HandleGlobalFlags() here:

https://github.com/buildkite/agent/blob/09bd9dbb76b23e986342685707751808e1c4da76/clicommand/global.go#L137-L147